integration/integration_test.go

```diff
@@ -468,7 +468,7 @@ func TestExecsnoop(t *testing.T) {
 	execsnoopCmd := &command{
 		name:           "StartExecsnoopGadget",
 		cmd:            fmt.Sprintf("$KUBECTL_GADGET trace exec -n %s", ns),
-		expectedRegexp: fmt.Sprintf(`%s\s+test-pod\s+test-pod\s+date`, ns),
+		expectedRegexp: fmt.Sprintf(`%s\s+test-pod\s+test-pod\s+\d+\s+\d+\s+date`, ns),
```

Comment:
Could you please describe why these changes are needed? Do we need to update some documents as well?